packages/app-store/templates/audit-log-implementation/components/NavigationPanel.tsx

```diff
@@ -0,0 +1,8 @@
+import { VerticalTabs } from "@calcom/ui";
```

Comment:
Since the app interface can be quite complex, this component is meant to enable easy navigation between all available panels. Its also easily extendable, getAppSettingsOptions can be altered to add/delet tabs when panels are added/removed